applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js

```diff
@@ -107,67 +107,73 @@ return view.extend({
 		o.rmempty  = false;
 
 		s.taboption('general', form.Flag, 'enable_upnp', _('Enable UPnP IGD protocol')).default = '1'
-		s.taboption('general', form.Flag, 'enable_natpmp', _('Enable PCP/NAT-PMP protocol')).default = '1'
+		s.taboption('general', form.Flag, 'enable_pcp_pmp', _('Enable PCP/NAT-PMP protocols')).default = '1'
 
-		s.taboption('general', form.Flag, 'secure_mode', _('Enable secure mode'),
-			_('Allow adding port forwards only to requesting IP addresses')).default = '1'
+		o = s.taboption('general', form.ListValue, 'upnp_igd_compat', _('UPnP IGD compatiblity mode'))
+		o.depends('enable_upnp', '1')
+		o.value('igdv1',_('IGDv1 (IPv4 only)'))
+		o.value('igdv2',_('IGDv2'))
 
-		s.taboption('general', form.Flag, 'igdv1', _('Enable UPnP IGDv1 mode'),
-			_('Advertise as UPnP IGDv1 device (no IPv6) instead of IGDv2')).default = '0'
+		o = s.taboption('general', form.Value, 'download_kbps', _('Download speed'),
+			_('Value in kbit/s, informational only'))
+		o.depends('enable_upnp', '1')
+		o.rmempty = true
 
-		s.taboption('general', form.Flag, 'log_output', _('Enable additional logging'),
-			_('Puts extra debugging information into the system log'))
+		o = s.taboption('general', form.Value, 'upload_kbps', _('Upload speed'),
+			_('Value in kbit/s, informational only'))
+		o.depends('enable_upnp', '1')
+		o.rmempty = true
 
-		s.taboption('general', form.Value, 'download', _('Download speed'),
-			_('Value in KByte/s, informational only')).rmempty = true
+		s.taboption('advanced', form.Flag, 'secure_mode', _('Enable secure mode'),
+			_('Allow adding port maps only to requesting IP addresses and deny third-party PCP option')).default = '1'
 
-		s.taboption('general', form.Value, 'upload', _('Upload speed'),
-			_('Value in KByte/s, informational only')).rmempty = true
+		o = s.taboption('advanced', form.Value, 'notify_interval', _('Notify interval'), _('A 900s interval will result in %s notifications with the minimum max-age of 1800s', 'A 900s interval will result in %s (%s = SSDP) notifications with the minimum max-age of 1800s').format('<abbr title="Simple Service Discovery Protocol">SSDP</abbr>'))
+		o.depends('enable_upnp', '1')
+		o.datatype    = 'uinteger'
+		o.placeholder = 900
 
-		o = s.taboption('general', form.Value, 'port', _('Port'))
+		o = s.taboption('advanced', form.Value, 'port', _('SOAP/HTTP port'))
+		o.depends('enable_upnp', '1')
 		o.datatype = 'port'
 		o.default  = 5000
 
-		s.taboption('advanced', form.Flag, 'system_uptime', _('Report system instead of service uptime')).default = '1'
-
-		s.taboption('advanced', form.Value, 'uuid', _('Device UUID'))
-		s.taboption('advanced', form.Value, 'serial_number', _('Announced serial number'))
-		s.taboption('advanced', form.Value, 'model_number', _('Announced model number'))
+		o = s.taboption('advanced', form.Value, 'presentation_url', _('Presentation URL'))
```

Comment:
Elaborate on what a presentation URL is for users encountering this setting for the first time.